### PR TITLE
Add `chainLogoUri` field to `Chain` model

### PR DESCRIPTION
**Pull request by iamacook**
This adds a new `chainLogoUri` `ImageField` to the `Chain` model which returns an absolute URL to the uploaded image, as well as adding/updating associated tests.

The implementation matches that of `currency_logo_uri` as [has been confirmed by the clients](https://github.com/safe-global/safe-config-service/issues/1021#issuecomment-1927023287):

- PNG support (max 512x512px, unrestricted file size)
- Uploaded as `*chain_logo.png`

### CODE REVIEW

File: src/chains/tests/test_models.py [hunk 1]
```diff
@@ -18,6 +18,13 @@ def test_str_method_outputs_name_chain_id(self) -> None:
         )
 
 
+class ChainLogoTestCase(TestCase):
+    def test_chain_logo_upload_path(self) -> None:
+        chain = ChainFactory.create(id=69)
+
+        self.assertEqual(chain.chain_logo_uri.url, "/media/chains/69/chain_logo.jpg")
```
Comment (fmrsabino) [inline src/chains/tests/test_models.py:25]: You can use the generated `chain.id` (generated by Faker) to validate the path also (to cover random chain ids).

File: src/chains/tests/test_views.py [hunk 2]
```diff
@@ -295,6 +301,29 @@ def test_null_ens_registry_address(self) -> None:
         self.assertEqual(response.json()["ensRegistryAddress"], None)
 
 
+class ChainLogoTests(APITestCase):
```
Comment (fmrsabino) [inline src/chains/tests/test_views.py:304]: These tests should be moved to `src/chains/tests/test_models.py` as we are not testing the view directly here.

Also extending `TestCase` should be enough for these tests.

**Review by fmrsabino — CHANGES_REQUESTED** [review]

File: src/chains/tests/test_models.py (hunk 1 above)
Comment (iamacook) [reply src/chains/tests/test_models.py]: I adjusted it in 82083887dcd88a0e34c9b70fe269a468686c90e2, along with that of the `currency_logo` for consistency.

File: src/chains/tests/test_views.py (hunk 2 above)
Comment (iamacook) [reply src/chains/tests/test_views.py]: I moved it in 82083887dcd88a0e34c9b70fe269a468686c90e2, along with that of the `currency_logo` for consistency.

**Review by fmrsabino — APPROVED** [review]
🚀

**Pull request merged**